React/Libraries.md

```diff
@@ -0,0 +1,53 @@
+#### React Libraries
+
+##### React Development Tools
+ - [create-react-app](https://github.com/facebook/create-react-app) - Project setup by running one command
+ - [storybook](https://github.com/storybookjs/storybook) - Development environment for UI components
+
+##### React Frameworks
+- [next.js](https://github.com/vercel/next.js) - The React Framework
+
```

Comment:
We need a section like Next plugins and put some common plugins that we use:
- next-compose-plugins
- next-react-svg
- next-seo

Reply:
Added :+1: